doc/examples/registration/plot_register_affine.py

```diff
@@ -0,0 +1,187 @@
+"""
+=================================================
+Estimate affine transformation to register images
+=================================================
+
+In this example, we use image registration to find an affine transformation
+which can be used to align our target and reference images.
+
+The :func:`skimage.measure.register_affine` function uses a Gaussian pyramid,
+a cost function, and an optimization function to find the affine transformation
+that best aligns a reference image to a target image. This transformation
```

Comment:
```suggestion
that best aligns a reference image to a moving image. This transformation
```
Isn't it the other way around?